connector/docker/README.md

```diff
@@ -0,0 +1,27 @@
+# Introduction
+
+This is sample project to demonstrate how to use the docker to run the connector.
+This examples is for http connector but can be used for any connector.
+
+# To build image
+
+To build docker image run:
+```
+make build
+```
+
+This will copy following files from your local file system:
+- your current fluvio profile
+- connector.yaml
+
+The connector.yaml is connector config for http source
```

Comment:
We should add a link to `connector.yaml` or inline it.

I ran without it... and got an error.